frontend/src/app/core/days/weekday.service.ts

```diff
@@ -38,13 +38,16 @@ import {
 import { InjectField } from 'core-app/shared/helpers/angular/inject-field.decorator';
 import { WeekdayResourceService } from 'core-app/core/state/days/weekday.service';
 import { IWeekday } from 'core-app/core/state/days/weekday.model';
-import { Observable } from 'rxjs';
+import {
+  Observable,
+  of,
+} from 'rxjs';
 
 @Injectable({ providedIn: 'root' })
 export class WeekdayService {
   @InjectField() weekdaysService:WeekdayResourceService;
```

Comment:
What's the difference between these two services? Can we reduce this to just one? Doesn't have to be in this PR, just wondering.

Reply:
I think the idea was to separate akita state from business logic, but they got intermixed with loading/caching as well. the caching should happen in the akita service. :+1: on unifying them